### PR TITLE
docs(grafana): add references to Grafana.com

### PR DESCRIPTION
**Pull request by jaellio**
Adds back references to Grafana.com. Reverts changes made in
(#280).

Signed-off-by: jaellio <jaellio@microsoft.com>

### CODE REVIEW

File: content/docs/demos/prometheus_grafana.md [hunk 1]
```diff
@@ -347,11 +347,13 @@ Select `Save and Test` and confirm you see `Data source is working`.
...
 
-OSM Dashboards are available through [OSM GitHub repository](https://github.com/openservicemesh/osm/tree/{{< param osm_branch >}}/charts/osm/grafana/dashboards), which can be imported as json blobs on the management application.
+OSM Dashboards are available from the following sources:
+- [OSM GitHub repository](https://github.com/openservicemesh/osm/tree/{{< param osm_branch >}}/charts/osm/grafana/dashboards)
+- [Online at Grafana.com](https://grafana.com/orgs/openservicemesh/dashboards)
 
 To import a dashboard:
 * Hover your cursor over the `+` and select `Import`.
-* Copy the JSON from the [osm-mesh-envoy-details dashboard](https://raw.githubusercontent.com/openservicemesh/osm/{{< param osm_branch >}}/charts/osm/grafana/dashboards/osm-mesh-envoy-details.json) and paste it in `Import via panel json`.
+* Copy the JSON from the [osm-mesh-envoy-details dashboard](https://raw.githubusercontent.com/openservicemesh/osm/{{< param osm_branch >}}/charts/osm/grafana/dashboards/osm-mesh-envoy-details.json) and paste it in `Import via panel json`. Alternatively, you can enter the dashboard's ID from `Grafana.com`. For example, the `osm-mesh-envoy-details` dashboard uses id `14145`.
```
Comment (zr-msft) [inline content/docs/demos/prometheus_grafana.md:356]:
```suggestion
* Copy the JSON from the [osm-mesh-envoy-details dashboard](https://raw.githubusercontent.com/openservicemesh/osm/{{< param osm_branch >}}/charts/osm/grafana/dashboards/osm-mesh-envoy-details.json) and paste it in `Import via panel json`. Alternatively, you can enter the dashboard's ID from [Grafana Dashboards](https://grafana.com/grafana/dashboards/). For example, the [OSM Mesh Details](https://grafana.com/grafana/dashboards/14145) dashboard uses ID `14145`.
```

File: content/docs/demos/prometheus_grafana.md [hunk 2]
```diff
@@ -347,11 +347,13 @@ Select `Save and Test` and confirm you see `Data source is working`.
 
 ## Importing OSM Dashboards
 
-OSM Dashboards are available through [OSM GitHub repository](https://github.com/openservicemesh/osm/tree/{{< param osm_branch >}}/charts/osm/grafana/dashboards), which can be imported as json blobs on the management application.
+OSM Dashboards are available from the following sources:
+- [OSM GitHub repository](https://github.com/openservicemesh/osm/tree/{{< param osm_branch >}}/charts/osm/grafana/dashboards)
+- [Online at Grafana.com](https://grafana.com/orgs/openservicemesh/dashboards)
```
Comment (zr-msft) [inline content/docs/demos/prometheus_grafana.md:352]:
```suggestion
- [OSM Grafana Dashboards](https://grafana.com/orgs/openservicemesh/dashboards)
```

**Comment by bridgetkromhout** [comment]
Note that when you merge this PR, you'll also want to cherry pick it into any release branches where it's relevant.

**Comment by zr-msft** [comment]
@jaellio were you still interested in getting this merged? if so can you review my requested changes?

**Comment by jaellio** [comment]
I added the do-not-merge/hold flag to this PR since the release guide hasn't been updated to include instructions for uploading and versioning the dashboards. If others are okay with this PR being merged without prior updates to the release guide I can remove the flag. I believe the instructions had previously been in the OSM release guide but were removed to reduce the complexity/noise in the release guide.

**Comment by jaellio** [comment]
Closing this issue for now since there has been no active discussion around updating the release guide. We should consider in the future automating the process of uploading Grafana dashboards to simplify the release process. Until we have committed to updating the dashboards in the Grafana.com repository the changes in this PR should not be merged.

**Pull request closed without merging**